src/components/Home.vue

```diff
@@ -105,12 +107,14 @@ function getNextTask() {
           <v-card-action
             class="d-flex justify-center justify-space-around flex-row"
           >
```

Comment:
This isn't relevant to this particular PR, but the alignment of those buttons is bothering me.
If you add `align-center` and `mb-3` here, and remove `my-2` from the "Skip to Next" button, they'll line up better in this space.
You can also remove `justify-center` as its overwritten by `justify-space-around`